Standards/scs-0104-v1-standard-images.md

```diff
@@ -103,7 +103,8 @@ recommended, while only Ubuntu 22.04 is mandatory.
 
 ### YAML lifecycle
 
-The YAML file is generally located in this repository under `/Tests/iaas`.
+The YAML file is generally located under
+<https://github.com/SovereignCloudStack/standards/blob/main/Tests/iaas/>.
```

Comment:
I'm not familiar with this `<` and `>` syntax for links in Markdown. It seems to render fine using "View file" here on GH and passes the PR checks but I could not find this links syntax at:

- https://www.markdownguide.org/basic-syntax/#links
- https://docs.gitlab.com/ee/user/markdown.html#links
- https://docusaurus-archive-october-2023.netlify.app/docs/markdown-features/links

I'm fine with it in case it works on the rendered docs.scs.communit page as well, I'm just curious which Markdown flavor this belongs to.

Reply:
Oh, that's a good question. I honestly don't know, but: I think I started doing it because our very own linter demanded it!

Reply:
It works with HedgeDoc, Github, Docusaurus and, AFAICT with the Python markdown package.

Reply:
Found it: https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses

URL was the keyword, not "link" ;)

Reply:
Never used that before, TIL.